README.md

```diff
@@ -141,7 +141,7 @@ swiftgen images /dir/to/search/for/imageset/assets
 This will generate an `enum Asset` with one `case` per image asset in your assets catalog, so that you can use them as constants.
 
 <details>
-<summary>Example of **code generated by the default template**</summary>
+<summary>Example of code generated by the default template</summary>
```

Comment:
I think this was meant as markdown (bold). Seems github doesn't mix HTML tags with markdown. Looks ok to me without the *, and bold might be a bit much tbh.